README.md

```diff
@@ -42,11 +42,9 @@ All config is in sam.[ENVIRONMENT].yml templates, encrypted as necessary.
 
 The following will invoke the lambda against the sample event jsons:
 ```
-sam local invoke --event event.[bib/item].json --region us-east-1 --template sam.[ENVIRONMENT].yml --profile [aws profile]
+AWS_ACCESS_KEY_ID=[...] AWS_SECRET_ACCESS_KEY=[...] bundle exec sam local invoke --event event.[bib|item].json --region us-east-1 --template sam.local.yml
```

Comment:
Let's investigate why you can't use `--profile` here

Reply:
Actually with the changes to `kms_client.rb` now I think it should work